.gitignore

```
@@ -20,4 +20,7 @@ anoma-*.tar.gz
 .python-version
 
 # generated if using IntelliJ IDEs
-/.idea
\ No newline at end of file
+/.idea
+
+# wasm artifacts
+*.wasm
```

let's not make this global, there are some .wasm files checked in in `wasm_for_tests` and we might want to add more. Maybe:
```suggestion
wasm/*.wasm
```